common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -747,6 +749,43 @@
                     self.isMouseFocus = false;
                     self.rendered = true;
                     self.state.el.addClass('is-captions-rendered');
+
+                    HtmlUtils.setHtml(
+                        self.subtitlesEl.find('.subtitles-menu').html,
+                        HtmlUtils.joinHtml(
+                            HtmlUtils.HTML('<li>'),
+                            gettext('Transcript will be displayed when you start playing the video.'), // jshint ignore:line
+                            HtmlUtils.HTML('</li>')
+                        )
+                    );
+
+                    self.subtitlesEl
+                        .attr('aria-label', gettext('Video transcript: activating an item in this group will spool the video to the corresponding time point.')); // jshint ignore:line
+
+                    HtmlUtils.setHtml(
```

Comment:
@clrux when you are setting text that is not HTML, I confirmed with @andy-armstrong that the best practice is to use JQuery's .text method (instead of HtmlUtils.setHtml).